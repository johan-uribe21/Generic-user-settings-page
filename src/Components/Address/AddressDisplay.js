import React from 'react';

//  stateless display component
const AddressDisplay = ({address}) => {

    return(
      <div className="beam-text" id='addressDisplay'>
        <span className="beam-text " id="street">
          {address.street}
        </span><br/>
        <span className="beam-text" id="city">
          {address.city} 
        </span><br/>
        <span className="beam-text " id="state">
          {address.state} 
        </span>
        <br/>
        <span className="beam-text" id="zipcode" >
          {address.zipcode} 
        </span><br/><br/>
        <input 
            type="submit"
            className='beam-edit-button' 
            value='Update'/>
        </div>
    )
  
}

export default AddressDisplay;