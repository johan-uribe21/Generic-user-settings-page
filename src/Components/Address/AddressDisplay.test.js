import {shallow, configure} from 'enzyme';
import AddressDisplay from './AddressDisplay';
import React from 'react';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() }); 

const mockAddress =  {
    street: '123 elm',
    city: 'cbus',
    state: 'OH',
    zipcode: '43023'
  }
  

const wrapper = shallow(<AddressDisplay 
    address={mockAddress}/>)

it('renders the address display component', () => {
  expect(wrapper).toMatchSnapshot();
});
