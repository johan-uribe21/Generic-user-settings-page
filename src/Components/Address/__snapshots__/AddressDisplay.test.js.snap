// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the address display component 1`] = `ShallowWrapper {}`;
